recipes/freetype/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+            --with-zlib=$PREFIX \
+            --with-png=yes \
+            --with-bzip2=no \
+            --without-harfbuzz
```

Comment:
Go for it, @ocefpaf! Here is one [recipe]( https://github.com/conda/conda-recipes/tree/master/harfbuzz ) to start with.